nnpdf_data/nnpdf_data/new_commondata/STAR-2006_SHP_200GEV/metadata.yaml

```diff
@@ -0,0 +1,51 @@
+# Generalia
+setname: "STAR-2006_SHP_200GEV"
+
+version: 1
+version_comment: "Initial implementation"
+
+# References
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1253360"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1253360"
+  version: 1
+
+nnpdf_metadata:
+  nnpdf31_process: "SHP" # Single Hadron Production
+  experiment: "STAR"
+
+implemented_observables:
+  - observable_name: "ALL"
+    observable:
+      description: "Neutral pion spin asymmetries at intermediate pseudorapidity in polarized proton collisions sqrt(s)=200 GeV"
+      label: "$A_{LL$"
```

Comment:
```suggestion
      label: "$A_{LL}$"
```

Reply:
Done in [5af0d56](https://github.com/NNPDF/nnpdf/pull/2154/commits/5af0d563f28adaa864d3620182bb7c7d58c88e77).